client/src/index.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import $ from 'jquery';
+import styled from 'styled-components';
+import Product from './components/Product.jsx';
+import Styles from './components/Styles.jsx';
+
+const ProductInfo = styled.div`
+  display: flex;
+  flex-flow: column;
+`;
+
+class ProductModule extends React.Component {
+
+  constructor(props) {
+    const ids = window.location.pathname.split('/');
+    super(props);
+    this.state = {
+      productId: ids[2],
+      styleId: ids[3],
```

Comment:
Pull out into variable to be able to validate data type, try catch with parseInt, and set a sensible default